htsohm/simulation/helium_void_fraction.py

```diff
@@ -22,28 +22,25 @@ def write_raspa_file(filename, uuid, simulation_config):
     Writes RASPA input-file.
 
     """
-    simulation_cycles = simulation_config['simulation_cycles']
+    # Load simulation parameters from config
+    values = {
+            'NumberOfCycles'                : params['simulation_cycles'],
+            'FrameworkName'                 : uuid}
+
+    # Load template and replace values
+    htsohm_dir = os.path.dirname(htsohm.__file__)
+    input_template_path = os.path.join(htsohm_dir, 'simulation', 'helium_void_fraction.input')
+    input_file = open(input_template_path)
+    template = Template( input_file.read() )
+    input_data = template.substitute(values)
+
```

Comment:
The above 5 lives of code look the same as above. You could write a helper function, something like:

load_and_subs_template(template_name, params)

That you could import from all use cases.